src/Compilers/CSharp/Test/Emit2/Semantics/RefReadonlyParameterTests.cs

```diff
@@ -6148,9 +6148,6 @@ public void Conversion_Lambda()
             delegate void O(out int p);
             """;
         CreateCompilation(source).VerifyDiagnostics(
-            // (7,6): error CS1661: Cannot convert lambda expression to type 'RR' because the parameter types do not match the delegate parameter types
-            // rr = (int x) => throw null;
-            Diagnostic(ErrorCode.ERR_CantConvAnonMethParams, "(int x) => throw null").WithArguments("lambda expression", "RR").WithLocation(7, 6),
```

Comment:
I didn't understand why these errors disappeared. Converting lambda with by-value argument to delegate with `ref readonly` parameter should still be illegal, right?

Reply:
It's still illegal, we just removed the overlapping errors from being reported on ref-kind mismatch. Notice directly below this line there lies an error in the same line for the parameter `x` specifically, which still considers the case illegal.